bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -706,4 +747,59 @@ private synchronized void handleCommandScheduled(ChannelUID channelUID, Command
                 new java.util.Date(thisBridge.lastSuccessfulCommunication()).toString());
         logger.trace("handleCommandScheduled({}) done.", Thread.currentThread());
     }
+
+    /**
+     * Register the exported actions
+     */
+    @Override
+    public Collection<Class<? extends ThingHandlerService>> getServices() {
+        return Collections.singletonList(VeluxActions.class);
+    }
+
+    /**
+     * Exported method (called by an OpenHAB Rules Action) to issue a reboot command to the hub.
+     *
+     * @return true if the command could be issued
+     */
+    public boolean runReboot() {
+        logger.trace("runReboot() called on {}", getThing().getUID());
+        RunReboot bcp = thisBridge.bridgeAPI().runReboot();
+        if (bcp != null && handleScheduler != null) {
+            // background execution of reboot process
+            handleScheduler.execute(() -> {
+                if (thisBridge.bridgeCommunicate(bcp)) {
+                    logger.info("Reboot command {}sucessfully sent to {}", bcp.isCommunicationSuccessful() ? "" : "un",
+                            getThing().getUID());
```

Comment:
See above. Logging to info.

Reply:
Again @fwolter I must disagree with you on this specific case. The case relates to the user issuing a remote reboot command to the Bridge. Under normal circumstances this would never need to be done. The command is only implemented because some users experience a lock up of the Bridge. So remote reboot is a really exceptional case, and therefore it must definitely be logged at 'INFO' (if not even higher..)